lib/elixir/lib/exception.ex

```diff
@@ -673,6 +675,7 @@ defmodule Exception do
       #=> "#Function<...>/1"
 
   """
+  @spec format_fa(fun, non_neg_integer) :: String.t()
```

Comment:
Should we use this here?

```suggestion
  @spec format_fa(fun, arity) :: String.t()
```